src/utils/headers.test.js

```diff
@@ -141,99 +141,35 @@ test('_headers: validate rules', async (t) => {
 
 test('_headers: headersForPath testing', async (t) => {
   const rules = await parseHeadersFile(t, '_headers')
-  const normalizedHeaders = rules.map(normalizeHeader)
-  t.deepEqual(headersForPath(normalizedHeaders, '/'), {
+  t.deepEqual(headersForPath(rules, '/'), {
     'X-Frame-Options': 'SAMEORIGIN',
     'X-Frame-Thing': 'SAMEORIGIN',
   })
-  t.deepEqual(headersForPath(normalizedHeaders, '/placeholder'), {
+  t.deepEqual(headersForPath(rules, '/placeholder'), {
     'X-Frame-Thing': 'SAMEORIGIN',
   })
-  t.deepEqual(headersForPath(normalizedHeaders, '/static-path/placeholder'), {
+  t.deepEqual(headersForPath(rules, '/static-path/placeholder'), {
     'X-Frame-Thing': 'SAMEORIGIN',
     'X-Frame-Options': 'DENY',
     'X-XSS-Protection': '1; mode=block',
     'cache-control': 'max-age=0, no-cache, no-store, must-revalidate',
   })
-  t.deepEqual(headersForPath(normalizedHeaders, '/static-path'), {
+  t.deepEqual(headersForPath(rules, '/static-path'), {
     'X-Frame-Thing': 'SAMEORIGIN',
     'X-Frame-Options': 'DENY',
     'X-XSS-Protection': '1; mode=block',
     'cache-control': 'max-age=0, no-cache, no-store, must-revalidate',
   })
-  t.deepEqual(headersForPath(normalizedHeaders, '/placeholder/index.html'), {
+  t.deepEqual(headersForPath(rules, '/placeholder/index.html'), {
     'X-Frame-Options': 'SAMEORIGIN',
     'X-Frame-Thing': 'SAMEORIGIN',
   })
-  t.deepEqual(headersForPath(normalizedHeaders, '/placeholder/_next/static/chunks/placeholder'), {
+  t.deepEqual(headersForPath(rules, '/placeholder/_next/static/chunks/placeholder'), {
     'X-Frame-Thing': 'SAMEORIGIN',
     'cache-control': 'public, max-age=31536000, immutable',
   })
-  t.deepEqual(headersForPath(normalizedHeaders, '/directory/placeholder/test.html'), {
+  t.deepEqual(headersForPath(rules, '/directory/placeholder/test.html'), {
     'X-Frame-Thing': 'SAMEORIGIN',
     'X-Frame-Options': 'test',
   })
 })
-
-/**
- * The bulk of the _headers logic concerns testing whether or not a path matches
- * a rule - focus on testing `matchesPath` over `headersForPath` (the latter just
- * straightforwardly combines a bunch of objects).
- */
-test('_headers: matchesPath matches rules as expected', (t) => {
```

Comment:
Those tests are redundant with `netlify-headers-parser` tests.
Since those are unit tests and not integration tests, they do not make much sense anymore.